back/app/models/custom_field.rb

```diff
@@ -4,36 +4,40 @@
-#  hidden                        :boolean          default(FALSE), not null
-#  maximum                       :integer
-#  logic                         :jsonb            not null
-#  answer_visible_to             :string
-#  select_count_enabled          :boolean          default(FALSE), not null
-#  maximum_select_count          :integer
-#  minimum_select_count          :integer
-#  random_option_ordering        :boolean          default(FALSE), not null
-#  page_layout                   :string
-#  linear_scale_label_1_multiloc :jsonb            not null
-#  linear_scale_label_2_multiloc :jsonb            not null
-#  linear_scale_label_3_multiloc :jsonb            not null
-#  linear_scale_label_4_multiloc :jsonb            not null
-#  linear_scale_label_5_multiloc :jsonb            not null
-#  linear_scale_label_6_multiloc :jsonb            not null
-#  linear_scale_label_7_multiloc :jsonb            not null
-#  dropdown_layout               :boolean          default(FALSE), not null
+#  id                             :uuid             not null, primary key
+#  resource_type                  :string
+#  key                            :string
+#  input_type                     :string
+#  title_multiloc                 :jsonb
+#  description_multiloc           :jsonb
+#  required                       :boolean          default(FALSE)
+#  ordering                       :integer
+#  created_at                     :datetime         not null
+#  updated_at                     :datetime         not null
+#  enabled                        :boolean          default(TRUE), not null
+#  code                           :string
+#  resource_id                    :uuid
+#  hidden                         :boolean          default(FALSE), not null
+#  maximum                        :integer
+#  logic                          :jsonb            not null
+#  answer_visible_to              :string
+#  select_count_enabled           :boolean          default(FALSE), not null
+#  maximum_select_count           :integer
+#  minimum_select_count           :integer
+#  random_option_ordering         :boolean          default(FALSE), not null
+#  page_layout                    :string
+#  linear_scale_label_1_multiloc  :jsonb            not null
+#  linear_scale_label_2_multiloc  :jsonb            not null
+#  linear_scale_label_3_multiloc  :jsonb            not null
+#  linear_scale_label_4_multiloc  :jsonb            not null
+#  linear_scale_label_5_multiloc  :jsonb            not null
+#  linear_scale_label_6_multiloc  :jsonb            not null
+#  linear_scale_label_7_multiloc  :jsonb            not null
+#  dropdown_layout                :boolean          default(FALSE), not null
+#  linear_scale_label_8_multiloc  :jsonb            not null
+#  linear_scale_label_9_multiloc  :jsonb            not null
+#  linear_scale_label_10_multiloc :jsonb            not null
+#  linear_scale_label_11_multiloc :jsonb            not null
```

Comment:
Can't help but feel that this is getting unwieldy. Wonder if combining all these multilocs into to one field might be a good future refactoring idea (not for this PR though)